index.html

```diff
@@ -1371,6 +1389,67 @@ <h3 id="name-requesting-a-credential-usi">
 </div>
 </section>
 </div>
+<div id="usage-of-decentralized-identifiers">
+<section id="section-2.6">
+        <h3 id="name-usage-of-decentralized-iden">
+<a href="#section-2.6" class="section-number selfRef">2.6. </a><a href="#name-usage-of-decentralized-iden" class="section-name selfRef">Usage of Decentralized Identifiers</a>
+        </h3>
+<p id="section-2.6-1"><a href="https://w3c.github.io/did-core/">Decentralized identifiers</a> are a resolvable identifier to a set of statements about the [did subject]() including a set of cryptographic material (e.g public keys). Using this cryptographic material, a <a href="https://w3c.github.io/did-core/">decentralized identifier</a> can be used as an authenticatable identifier in a credential.<a href="#section-2.6-1" class="pilcrow">¶</a></p>
```

Comment:
should did subject link to the definition of did subject in the did-core spec?

Reply:
Yeap it should will fix